### ISSUE TITLE
When using as typeof X it imports X in lua.

### ISSUE THREAD

**Issue by TrafficConeGod** [roblox-ts/roblox-ts#300]
When I say import X and only use it in as typeof X it compiles the import of X into lua (even though it's a type thing).

**Comment by Validark** [comment]
Can you give a code example?

**Comment by TrafficConeGod** [comment]
Sure. (btw this is like a very bad use case for as typeof X)

SomeClass.ts
```ts
export class SomeClass {

    

}
```

Main ts file
```ts
import { SomeClass } from "./SomeClass"

class OtherClass {

    

}

const Val = OtherClass as typeof SomeClass
const val = new Val()
```

The `import { SomeClass } from "./SomeClass"` compiles into lua despite the import only being used a s a type.

**Comment by evaera** [comment]
Closed in favor of #260
